jupiterone/questions/questions.yaml

```diff
@@ -1,15 +1,18 @@
 ---
-sourceId: managed:template
+sourceId: managed:probely
 integrationDefinitionId: '${integration_definition_id}'
-questions: []
-  # - id: integration-question-template-replace-me
-  #   title: What kinds of questions will this integration support?
-  #   description:
-  #     TODO Every integration should contribute questions! Please be carefule
-  #     to replace this before deploying the integration to JupiterOne.
-  #   queries:
-  #     - name: good
-  #       query: |
-  #         find * with _integrationDefinitionId = '${integration_definition_id}'
-  #   tags:
-  #     - template
\ No newline at end of file
+questions: 
+- id: managed-question-probely-common-vulnerability-scoring-findings-that-are-not-fixed
+  title: What are Probely's CVSS findings that have not been fixed?
+  description: 
+    Returns a list of Probely CVSS findings that are not fixed.
+  queries:
+    - name: Probely CVSS Findings That Require Remediation 
+      resultsAre: BAD
+      query: |
+        FIND UNIQUE probely_finding WITH state = 'notfixed' AS f
```

Comment:
Can you explain why `UNIQUE` is necessary here?

Reply:
I used UNIQUE because in my J1DEV environment the name creativeice.com is featured three times. Please refer to the attachment.
![Screen Shot 2022-09-12 at 2 58 40 PM](https://user-images.githubusercontent.com/93156050/189734534-b0fc9d62-41e2-4f9d-b9e9-4e4a21d218b6.png)

Reply:
@austinkelleher removed UNIQUE keyword.
Please review the change. TIA